app/components/Avatar.tsx

```diff
@@ -44,7 +44,7 @@ export const Avatar: React.FC<AvatarProps> = (props) => {
           key={`panel-${index}-${source}`}
           source={source}
           style={[$imageStyle, $panelImageStyle]}
-          resizeMode="contain"
+          resizeMode="cover"
```

Comment:
I think due to the dimension of the images in the cms for the guest speakers being rectangular whereas IR peeps images are square, the avatars in the speaker panel are broken ( I think they recently updated the images of the guests?) so I switched the resize mode to "cover"